corehq/apps/hqcase/export.py

```diff
@@ -57,10 +56,7 @@ def get_matching_owner(case):
             return get_owner_id(case)
 
     def might_be_relevant(case):
-        if filter_group:
-            return get_owner_id(case) in owner_ids
-        else:
-            return not users or get_matching_owner(case)
+        return not filter_group or get_owner_id(case) in owner_ids
```

Comment:
This is the same as

```python
         if filter_group:
            return get_owner_id(case) in owner_ids
        else:
            return True
```

right? What's was the logic behind this previous code that's now being changed?